### DISCUSSION TITLE
Matteo Collina BOT

### DISCUSSION THREAD

**Discussion by simoneb** [fastify/fastify#4307] [Ideas]
Have you considered creating a GitHub App that posts a comment on opened issues with @mcollina's usual motive?

> Thanks for reporting! Would you like to send a Pull Request to address this issue? Remember to add unit tests.

Imagine how much of Matteo's time this would save! ;)

**Comment by jsumners** [comment]
Heh. I just have a https://docs.github.com/en/get-started/writing-on-github/working-with-saved-replies/about-saved-replies

**Comment by Eomm** [comment]
It is a pun for sure, but it deserves a motivational answer:

it would become noisy, and the user may feel uncomfortable with such an answer.
When Matteo writes that sentence, I'm sure he has read the message, and there is not much more to say

Triage is always a pain

**Comment by mcollina** [comment]
I can see if I can use a pat to do it for myself on certain repos.
